src/static/swagger.json

```diff
@@ -788,15 +879,132 @@
+                "Ed25519VerificationKey2020",
+                "JsonWebKey2020"
+              ]
+            }
+          },
+          {
+            "in": "query",
+            "name": "service",
+            "description": "Redirects to Service Endpoint.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "query",
+            "name": "relativeRef",
+            "description": "Addition to Service Endpoint.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "query",
+            "name": "metadata",
+            "description": "Show only metadata of DID Document.",
+            "schema": {
+              "type": "boolean"
+            }
+          },
+          {
+            "in": "query",
+            "name": "resourceId",
+            "description": "Filter by ResourceId.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "query",
+            "name": "resourceCollectionId",
+            "description": "Filter by CollectionId.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "query",
+            "name": "resourceType",
+            "description": "Filter by Resource Type.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "query",
+            "name": "resourceName",
+            "description": "Filter by Resource Name.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "query",
+            "name": "resourceVersion",
+            "description": "Filter by Resource Version.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "query",
+            "name": "resourceVersionTime",
+            "description": "Get the nearest resource by creation time.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "in": "query",
+            "name": "resourceMetadata",
+            "description": "Show only metadata of resources.",
+            "schema": {
+              "type": "boolean"
+            }
+          },
+          {
+            "in": "query",
+            "name": "checksum",
+            "description": "Sanity check that Checksum of resource is the same as expected.",
+            "schema": {
+              "type": "string"
+            }
           }
         ],
         "responses": {
           "200": {
             "description": "The request was successful.",
             "content": {
-              "application/json": {
+              "any": {
                 "schema": {
-                  "$ref": "#/components/schemas/DidDocument"
+                  "type": "object"
```

Comment:
This should probably still be DID Document? At least as default

Reply:
Yes please because that's usually the default, we can add a description/comment that if using query parameters the response might be a DID Document or a query/dereferencing result.

Reply:
Done! ✅